### ISSUE TITLE
import problem

### ISSUE THREAD

**Issue by bolz213** [ShuhuaGao/geppy#2]
Hi. I installed geppy following the instructions in readme, however if I cd to another project folder I can't import geppy. The error is 

`
ModuleNotFoundError: No module named 'geppy.core'
`

**Comment by ShuhuaGao** [comment]
After installation, it can be imported anywhere. Could you assure that it is installed properly? For example, in your terminal type `pip list` and you should see `geppy` in the list.  If it is true, then in your program can you run `import geppy` successfully?

**Comment by bolz213** [comment]
1. geppy installed:
<img width="382" alt="image" src="https://user-images.githubusercontent.com/9564786/51451860-832dfe80-1d8b-11e9-8074-43f99474bf7c.png">

2. under the dir of geppy I can import successfully:
<img width="569" alt="image" src="https://user-images.githubusercontent.com/9564786/51451900-ae185280-1d8b-11e9-9882-8410b47751cf.png">

3. cd to another dir, error happens:
<img width="570" alt="image" src="https://user-images.githubusercontent.com/9564786/51451924-d011d500-1d8b-11e9-909a-a86624a4fa04.png">

Currently I copy geppy files to my project dir so that I can use geppy. But it will be more convenient if you can fix this. Thanks!

**Comment by ShuhuaGao** [comment]
Yes, you are right. The error can be reproduced on Ubuntu, but not on Windows. I did't find the reason yet.  However, a temporary solution is:
-  first `pip uninstall geppy` on your system.
- then, in the *geppy* directory install it via `pip install -e .` (the editing mode). 

I have tested it on Ubuntu. It should work now.
